### PR TITLE
Updation in index file

### LINKED ISSUE #1080: Updation in index file
image

image

removal of one from above two as both contains same content ,CSS JS code along with HTML in about section.

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#1080

## Related Issues

- Closes #1080 

## Type of PR

- [X] (feature)

## Screenshots / videos (if applicable)

image

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

File: Css-Files/aboutus.css [hunk 1]
```diff
@@ -85,74 +85,4 @@
...
-  .about-us-location,
-  .about-us-product {
-    width: 98% !important;
-  }
-
-  .about-us-blk-para {
-    font-size: 15px !important;
-  }
-}
+}
```
Comment (varshith257) [inline Css-Files/aboutus.css:88]:
```suggestion
}

```

File: index.html [hunk 2]
```diff
@@ -221,66 +221,7 @@ <h1 class="about-us-heading">About Us</h1>
...
-                </div>
-                <p class="about-us-blk-para">Rapidoc is dedicated to offering personalized healthcare services tailored to meet your unique needs. Our platform connects you with a variety of specialists and healthcare providers, ensuring you receive the best possible care. From tailored treatment plans to ongoing health management, Rapidoc supports your journey to optimal health with expert guidance and personalized attention.</p>
-              </div>
-              <div class="card-back">
-                <img src="./images/serviceshover.png" alt="Services Hover Image" style="width: 100%; height: 100%;">
-              </div>
-            </div>
-          </div>
-        </div>
+
```
Comment (varshith257) [inline index.html:224]:
```suggestion
```

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Pull request merged**